src/uast.cc

```diff
@@ -53,49 +73,109 @@ void *NodeAt(const Nodes *nodes, int index) {
   if (index < nodes->len) {
     return nodes->results[index];
   }
-  return NULL;
+  return nullptr;
```

Comment:
Not sure if it's totally safe to return this here, since this is part of the C api.

Reply:
Should be safe if compiled with the C++ compiler; it'll replace the value by 0 or false depending on the context, and check that you don't cast it to/from integer and other things.